### PR TITLE
Package ui to `oxo serve`

### PR DESCRIPTION
**Pull request by mohsinenar**
Package ui to oxo serve

### CODE REVIEW

File: src/ostorlab/serve_app/app.py [hunk 1]
```diff
@@ -58,7 +56,24 @@ def ubjson_decode(self, request: flask.Request):
...
-    def dispatch_request(self) -> flask.Response:
+    @staticmethod
+    def authenticate() -> Optional[tuple[flask.Response, int]]:
+        """Authenticate the request."""
+        if flask.request.method == "OPTIONS":
+            # CORS requires sending an initial OPTIONS that should return 200 OK.
+            return None
+
+        api_key = flask.request.headers.get(AUTHORIZATION_HEADER)
+        if api_key is None or models.APIKey.is_valid(api_key) is False:
```
[the remark this answers is not in the document]
Comment (mohsinenar) [reply src/ostorlab/serve_app/app.py:67]: The authenticate method is now inside the graphql view, wish mean it applied to it no matter what is the path.

File: src/ostorlab/ui/static_files/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>OXO UI</title>
+</head>
+<body>
+    <h1>Dummy</h1>
```
Comment (3asm) [inline src/ostorlab/ui/static_files/index.html:10]: Dummy?

**Review by 3asm — APPROVED** [review]

File: src/ostorlab/serve_app/app.py [hunk 3]
```diff
@@ -25,19 +27,15 @@ def create_app(path: str = "/graphql", **kwargs) -> flask.Flask:
...
+        """Serve the Ui folder"""
+        return flask.send_from_directory(str(UI_STATIC_FILES_DIRECTORY), "index.html")
 
-        if api_key is None or models.APIKey.is_valid(api_key) is False:
-            return flask.jsonify({"error": "Unauthorized"}), 401
-        else:
-            # The request is authenticated. Continue.
-            return None
+    @app.route("/<path:file_path>")
+    def serve_static(file_path):
```
Comment (amine3) [inline src/ostorlab/serve_app/app.py:36]: Missing return type

File: src/ostorlab/ui/static_files/index.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<html>
```
Comment (amine3) [inline src/ostorlab/ui/static_files/index.html:1]: Is this the final version of index.html?

**Review by amine3 — DISMISSED** [review]

File: src/ostorlab/ui/static_files/index.html (hunk 4 above)
Comment (mohsinenar) [reply src/ostorlab/ui/static_files/index.html:1]: Was a dummy file, Removed it now.

**Review by 3asm — APPROVED** [review]

**Pull request merged**